hoomd/md/IntegratorTwoStep.cc

```diff
@@ -84,14 +84,10 @@ void IntegratorTwoStep::update(uint64_t timestep)
         }
     else
 #endif
-        {
-        // Update rigid body constituent particles in serial simulations.
-        updateRigidBodies(timestep + 1);
-        }
 
     // compute the net force on all particles
 #ifdef ENABLE_HIP
-    if (m_exec_conf->isCUDAEnabled())
+        if (m_exec_conf->isCUDAEnabled())
```

Comment:
Please revert these changes. The first will break other tests, the second is a stray formatting change.